README.md

```diff
@@ -43,6 +43,12 @@ secs = 3
 nanos = 0
 ```
 
+When psh running as daemon, the config located `/etc/psh/config.toml`.
+
+```toml
+component_args = ["path/to/test-get-os.wasm"]
+```

```

Comment:
component has two parameters: 1). path, 2). the parameters that are applied to this component.